guides/common/modules/ref_capsule-storage-requirements.adoc

```diff
@@ -15,8 +15,11 @@ endif::[]
 |Directory |Installation Size |Runtime Size
 ifdef::katello,satellite,orcharhino[]
 |/var/lib/pulp |1 MB |300 GB
-|{postgresql-lib-dir} |100 MB |10 GB
+|{postgresql-lib-dir} |100 MB |20 GB
 endif::[]
 |/usr |3 GB |Not Applicable
 |/opt/puppetlabs |500 MB |Not Applicable
 |====
+
+The size of the `postgres` database on {SmartProxy} can grow significantly with an increasing number of lifecycle environments, content views, or repositories that the {SmartProxy} is synchronizing.
```

Comment:
```suggestion
The size of the PostgreSQL database on your {SmartProxyServer} can grow significantly with an increasing number of lifecycle environments, content views, or repositories that are synchronized from your {ProjectServer}.
```

I'd use "SmartProxyServer" based on the next sentence where you compare ProjectServer with SmartProxyServer.

Unsure if "content views" or "Content Views".